lib/json_api_client/scope.rb

```diff
@@ -24,11 +26,12 @@ def includes(*tables)
     end
 
     def page(number)
+      number = [number.to_i, FIRST_PAGE_NUMBER].max
```

Comment:
I feel like we'd rather raise an ArgumentError instead of silently accepting/changing the user's input